### PR TITLE
Fix README for CentOS images.

### PR DESCRIPTION
**Pull request by jerboaa**
PR https://github.com/redhat-developer/s2i-dotnetcore/pull/137 missed this
as @tmds pointed out. This should fix it.

### CODE REVIEW

**Comment by jerboaa** [comment]
[skipci]

File: README.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ Image name structure
 
 ##### Structure: 1/2-3-4
```
Comment (tmds) [inline README.md]: fyi, before I added centos prefix, it said: dotnet/1-2-3

suggestion: we can get rid of the numbers, and include the runtime: `dotnet/<prefix>-<version>-[runtime-]<os>`

Comment (tmds) [reply README.md]: I don't have a strong preference, so feel free to keep it as-is.

**Comment by jerboaa** [comment]
This seems simpler, so I've updated it. Removed the `builder` in section "Base image". It's really there to stress the base stack, not whether it's a s2i builder or not.

**Comment by jerboaa** [comment]
@tmds Good to go?

**Comment by tmds** [comment]
:shipit:

**Comment by jerboaa** [comment]
Thanks for the review!

**Pull request merged**